prime-router/docs/observability/common-queries.md

```diff
@@ -0,0 +1,64 @@
+# Common Queries in KQL
+
+## General tips
+- Use the `Time range` selector in the Azure query dialog to specify what time range
+you want your results from.
+- The value of `null` for a receiver in the `ReportRouteEvent` means that report was not routed to any receiver
+
+### Which senders have sent a report
+```
+customEvents
+| where name == "ReportAcceptedEvent"
+| extend sender = tostring(customDimensions.sender)
+| distinct sender
+```
+
+### Which receivers have had a report routed to them
```

Comment:
How about for a particular topic, like `elr-elims`?

Reply:
Can add "optional" lines to each query
```
| where topic == "elr-elims" \\ optionally add to query to restrict the results to a particular topic
```